docs/docs/segment-ui5tooling.md

```diff
@@ -0,0 +1,59 @@
+---
+id: ui5tooling
+title: UI5 Tooling
+sidebar_label: UI5 Tooling
+---
+
+## What
+
+Display the active [UI5 tooling](https://sap.github.io/ui5-tooling/) version (global or local if present -
```

Comment:
please use bottom style links (see https://commonmark.org/help/tutorial/07-links.html)

Reply:
Noted, I'll correct this.